airflow/contrib/hooks/snowflake_hook.py

```diff
@@ -64,6 +66,33 @@ def _get_conn_params(self):
             "region": self.region or region or '',
             "role": self.role or role or '',
         }
+
+        """
+        If private_key_file is specified in the extra json, load the contents of the file as a private
+        key and specify that in the connection configuration. The connection password then becomes the
+        passphrase for the private key. If your private key file is not encrypted (not recommended), then
+        leave the password empty.
+        """
+        private_key_file = conn.extra_dejson.get('private_key_file', None)
+        if private_key_file is not None:
```

Comment:
This line can be `if private_key_file:`.

Reply:
fixed